### ISSUE TITLE
HTTP/2(gRPC) Connection Problems

### ISSUE THREAD

**Issue by maskofaction** [Kong/kong#6761]
https://github.com/Kong/kong/issues/6066#issue-648059388

Currently, we have installed and tested Kong version 2.3. However, the same issue occurred again.
HTTP/2 is used for all interworking. When a request message is sent to the outside through Kong, TCP connection is disconnected as frequently as Kong handles HTTP/1.1.
However, there are cases where this phenomenon does not occur in a specific service, but it does occur in most cases. The two services under test show little difference in behavior.
In other words, there is almost no difference in HTTP/2 Header or Body with the same pattern.
I really don't know why this is happening and what should I do.

![message-flow](https://user-images.githubusercontent.com/67627969/105317183-43fa4880-5c05-11eb-9950-060347bae95e.jpg)

_Originally posted by @maskofaction in https://github.com/Kong/kong/issues/6066#issuecomment-764440189_

**Comment by javierguerragiraldez** [comment]
closing this, continuing the original issue #6066
